src/main/java/org/jabref/logic/importer/fileformat/ACMPortalParser.java

```diff
@@ -227,4 +222,24 @@ public BibEntry parseBibEntry(String jsonStr) {
 
         return bibEntry;
     }
+
+    private List<Author> getAuthors(JsonArray authors) {
+        List<Author> jabrefAuthors = new ArrayList<>();
+
+        for (JsonElement author : authors) {
+            JsonObject authorJsonObject = author.getAsJsonObject();
+
+            String given = null;
+            String family = null;
+            if (authorJsonObject.has("given")) {
+                given = authorJsonObject.get("given").getAsString();
+            }
+            if (authorJsonObject.has("family")) {
+                family = authorJsonObject.get("family").getAsString();
+            }
+            Author jabrefAuthor = new Author(given, null, null, family, null);
+            jabrefAuthors.add(jabrefAuthor);
+        }
+        return jabrefAuthors;
+    }
```

Comment:
Wouldn't it be more comprehensible, if the return type of this method would be already AuthorList instead of List<Author>, since in the only place you use this method, you create an Authorlist of this List of Authors...?